.github/workflows/archive.yml

```
@@ -66,7 +66,7 @@ jobs:
           swift --version
           echo Release version: ${{ inputs.version }}
           echo SDK: ${{ matrix.sdk }}
-    - name: Archive for iOS 
+    - name: Archive for respective platform SDK
```

I think you can even do this here:
```suggestion
    - name: Archive for ${{ matrix.sdk }}
```